Sources/ChatImpl.swift

```diff
@@ -26,15 +26,18 @@ public final class ChatImpl {
 
   let chat: PubNubChat.ChatImpl
 
-  /// Initializes a new instance with the given chat and `PubNub` configurations.
+  /// Creates a new instance with the given chat and `PubNub` configurations.
   ///
   /// This initializer sets up the object using the provided chat configuration and `PubNub` configuration.
   /// After creating an instance, you must call the ``initialize(completion:)`` method before using the object.
   ///
   /// - Parameters:
   ///   - chatConfiguration: A configuration object of type ``ChatConfiguration`` that defines the chat settings
   ///   - pubNubConfiguration: A configuration object of type `PubNubConfiguration` that defines the `PubNub` settings
-  public init(chatConfiguration: ChatConfiguration, pubNubConfiguration: PubNubConfiguration) {
+  public init(
+    chatConfiguration: ChatConfiguration = ChatConfiguration(),
```

Comment:
Is order of params ok? The thing is that `pubNubConfiguration` is mandatory whereas `chatConfiguration` is optional. Shouldn't mandatory parameter come first?

Reply:
Indeed, I haven't noticed it before. Swift does not enforce this rule at the compiler level, but it's a widely followed practice. Would it be fine for you to change it while releasing a major version? Thus, we could avoid introducing any breaking changes right now